File: app/src/main/java/com/example/AppTaskFirst/MainActivity.java
package com.example.AppTaskFirst;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    TextView textView;
    EditText editText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        textView = findViewById(R.id.fignya);
        editText = findViewById(R.id.editTextText);



    }

    public void onBtnClick(View view) {
        String str = editText.getText().toString();
        String srti = "";
        textView.setText("Молодец!!!");
        char [] chrus = str.toCharArray();
        for(int i =0;i<chrus.length;i++){
            if(chrus[i]== 'a'|| chrus[i]=='A'|| chrus[i]== 'а'|| chrus[i]== 'А'){
                chrus[i]='o';
            }
            srti = srti + chrus[i];
        }
        editText.setText(srti);


    }
}

File: app/src/main/java/com/example/AppTaskFirst/MainActivity.kt
package com.example.AppTaskFirst

import androidx.appcompat.app.AppCompatActivity
import android.widget.TextView
import android.widget.EditText
import android.os.Bundle
import android.view.View
import com.example.AppTaskFirst.R

class MainActivity : AppCompatActivity() {
    var textView: TextView? = null
    var editText: EditText? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        textView = findViewById(R.id.fignya)
        editText = findViewById(R.id.editTextText)
    }

    fun onBtnClick(view: View?) {
        val str = editText!!.text.toString()
        var srti = ""
        textView!!.text = "Молодец!!!"
        val chrus = str.toCharArray()
        for (i in chrus.indices) {
            if (chrus[i] == 'a' || chrus[i] == 'A' || chrus[i] == 'а' || chrus[i] == 'А') {
                chrus[i] = 'o'
            }
            srti = srti + chrus[i]
        }
        editText!!.setText(srti)
    }
}